spec/unit/devtools/flay/scale/measure_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'spec_helper'
+require 'flay'
+
+module Devtools
+  module Flay
+    describe Scale, '#measure' do
+      subject(:measure) { instance.measure }
+
+      let(:minimum_mass) { 0        }
+      let(:files)        { [double] }
+      let(:instance) do
+        described_class.new(minimum_mass: minimum_mass, files: files)
+      end
+      let(:flay_masses) { { 0 => 5, 1 => 10 } }
+      let(:flay_hashes) do
+        {
+          0 => double('sexp array', size: 3),
+          1 => double('sexp array', size: 11)
+        }
+      end
+
+      let(:flay) do
+        instance_double(
+          ::Flay,
+          analyze: nil,
+          masses: flay_masses,
+          hashes: flay_hashes
```

Comment:
`analyze` instructs flay to build up its results.  I don't use the result currently.  I could make this more clear by using `allow` I guess